### PR TITLE
Fix | Gidk 123: Daybook page loader issue

### PR DESCRIPTION
**Pull request by shubhamkeshari**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts [hunk 1]
```diff
@@ -462,17 +462,17 @@ export class AccountAddNewDetailsComponent implements OnInit, OnChanges, AfterVi
             } else {
                 // statesEle.setDisabledState(false);
                 if (this.isIndia) {
-                    statesEle.forceClearReactive.status = true;
-                    statesEle.clear();
-                    gstForm.get('stateCode').patchValue(null);
-                    gstForm.get('state').get('code').patchValue(null);
+                    // statesEle.forceClearReactive.status = true;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts:465]: Should not be commented as it is used to reset the state after GST is reduced to length less than 2

File: apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts [hunk 2]
```diff
@@ -462,17 +462,17 @@ export class AccountAddNewDetailsComponent implements OnInit, OnChanges, AfterVi
...
+                    // gstForm.get('stateCode').patchValue(null);
+                    // gstForm.get('state').get('code').patchValue(null);
                 }
             }
         } else {
-                statesEle.forceClearReactive.status = true;
-                statesEle.clear();
-                gstForm.get('stateCode').patchValue(null);
-                gstForm.get('state').get('code').patchValue(null);
+                // statesEle.forceClearReactive.status = true;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts:472]: Should not be commented as it is used to reset the state after GST is reduced to empty value

File: apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts (hunk 1 above)
Comment (shubhamkeshari) [reply apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts:465]: ok got it 👍

**Comment by shubhamkeshari** [comment]
Suggested changes are Implemented and working properly.

File: apps/web-giddh/src/app/daybook/daybook.component.ts [hunk 3]
```diff
@@ -33,6 +33,7 @@ import { DaybookAdvanceSearchModelComponent } from './advance-search/daybook-adv
 })
 export class DaybookComponent implements OnInit, OnDestroy {
     public companyName: string;
+    public showLoader: boolean = false;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/daybook/daybook.component.ts:36]: Add JS Doc

**Comment by shubhamkeshari** [comment]
JS doc added

**Pull request merged**